www/js/suggestions/suggestions-controller.js

```diff
@@ -12,13 +12,33 @@ angular.module('starter.suggestions', ['ionic', 'ionic.contrib.ui.tinderCards',
     // $scope.suggestions.push(newCard);
   };
 
+  $scope.toggleGoToSuggestion = function(suggestionId) {
+    var buttonId = "#going-" + suggestionId;
+    var hasBeenTo = $(buttonId).val();
+    if (hasBeenTo == "false") {
+      $scope.goToSuggestion(suggestionId);
+      $(buttonId).val("true");
+    }
+    else {
+      $scope.doNotGoToSuggestion(suggestionId);
+      $(buttonId).val("false");
+    } 
+  };
+
   $scope.goToSuggestion = function(suggestionId) {
-    var buttonId = "#going-" + suggestionId
+    var buttonId = "#going-" + suggestionId;
```

Comment:
since `goToSuggestion()` and `doNotGoToSuggestion()` are only ever called by `toggleGoToSuggestion()` we can pass in the `buttonId` as an argument into these two functions as it is already defined in `toggleGoToSuggestion()`.